ambientweather2sqlite/server.py

```
@@ -42,13 +42,13 @@ def _send_json(self, data: dict, status: int = 200) -> None:
                 json_string = json.dumps(data, indent=2)
                 self.wfile.write(json_string.encode("utf-8"))
             except BrokenPipeError:
-                # Client disconnected before response was sent
-                pass
+                self.DATABASE.log_error("BrokenPipeError", "Client disconnected before response was sent")
```

Ruff (0.11.9)

Line too long (106 > 88)

(E501)